docs/overview.rst

```diff
@@ -0,0 +1,11 @@
+Overview
+========
+
+The `CEDAR Madrigal database <http://cedar.openmadrigal.org/>`_ houses several
+ground- and space-based data sets for instruments that provide upper
+atmospheric, scientific observations.
```

Comment:
```suggestion
atmospheric scientific observations.
```

Reply:
It's two adjectives before a noun so the comma is required.